android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/AndroidPayCheckoutSession.java

```diff
@@ -1,45 +1,63 @@
 package com.shopify.unity.buy;
 
+import com.google.android.gms.wallet.WalletConstants;
 import com.shopify.buy3.pay.PayCart;
+import com.shopify.buy3.pay.PayHelper;
+import com.shopify.unity.buy.models.MailingAddressInput;
 import com.shopify.unity.buy.models.PricingLineItems;
 import com.shopify.unity.buy.utils.AndroidLogger;
 import com.shopify.unity.buy.utils.ILogger;
 import com.unity3d.player.UnityPlayer;
 
 import java.io.IOException;
 
-public final class AndroidPayCheckoutSession {
+public final class AndroidPayCheckoutSession implements AndroidPaySessionCallbacks {
     private ILogger logger;
 
     public AndroidPayCheckoutSession() {
         this.logger = new AndroidLogger();
     }
 
-    public AndroidPayCheckoutSession(ILogger logger) {
+    AndroidPayCheckoutSession(ILogger logger) {
```

Comment:
This will be removed in the unit testing patch I have up. Since I'm using instrument tests for this I'm changing this back to using Android's `Log` class.